packages/browser/README.md

```diff
@@ -340,18 +320,37 @@ The following options are available for the BacktraceClientOptions passed when i
+// send as a string
+await client.send('This is a string!');
+
+// send as an Error
+await client.send(new Error('This is an Error!'));
+
+// as a BacktraceReport (string)
+await client.send(new BacktraceReport('This is a report with a string!'));
+
+// as a BacktraceReport (Error)
+await client.send(new BacktraceReport(new Error('This is a report with a string!')));
+```
+
+### Modify/skip error reports
+A BeforeSend event is triggered when an exception in the managed environment occurs to which you can attach a handler. You can use the BeforeSend event to scrub PII, or extend attributes or JSON object data based on data your application has at the time of exception. A report can be skipped sompletely by returning a null value.
+
+```ts
+  // Code sample
+```
+
+### Custom file/http handlers
 
-| Name                                                                                                                                            | Return Type     | Description                                                                       |
-| ----------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | --------------------------------------------------------------------------------- |
-| `addAttribute(attributes: Record<string, unknown>)`                                                                                             | void            | Add attributes to the BacktraceClient reports                                     |
-| `addAttachment(attachment: BacktraceAttachment)`                                                                                                | void            | Add an attachment to the BacktraceClient reports                                  |
-| `initialize(options: BacktraceClientOptions)`                                                                                                   | BacktraceClient | Initializes a new BacktraceClient (returns the same instance on subsequent calls) |
-| `builder(options: BacktraceClientOptions).build()`                                                                                              | BacktraceClient | (Advanced) Sets up a new BacktraceClient for reporting                            |
-| `send(data: BacktraceReport \| Error \| string, reportAttributes: Record<string, unknown> = {}, reportAttachments: BacktraceAttachment[] = [])` | `Promise<void>` | Asynchronously sends error data to Backtrace                                      |
-| `dispose`                                                                                                                                       | void            | Disposes the client                                                               |
+Custom handlers can be implemented to override BacktraceClient file and http operations. Overriding the default operations allows custom encryption for data at rest or in motion.
```

Comment:
In the database we do not have file operations. We do have it in node/react-native/electron/nest.js

Reply:
we can leave file operations - maybe someone would like to use a file system in browser by using local storage or something similar. We do not know it yet